src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiSelect/MultiSelect.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import React from 'react';
+import type {Element} from 'react';
+import type {SelectChildren} from '../Select/types';
+import {GenericSelect, Option} from '../Select';
+
+type Props = {
+    values: Array<string>,
+    label: string,
```

Comment:
The label should be is not general. Sometimes there will be all elements separated by commas, other times there will be a counter, ...